TMT Mac/Views/MoviePosterView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  MoviePosterView.swift
+//  TheatricalMovieTrailers
+//
+//  Created by Christoph Parstorfer on 15.10.20.
+//  Ported to macOS by Christoph Parstorfer on 24.12.21.
+//
+
+import SwiftUI
+
+struct MoviePosterView: View {
+    private let filmPosterAspectRatio = CGFloat(0.7063020214)
+    
+    @State private var reflectionDistance: CGFloat
+    @State private var onTap: (() -> ())?
+    @Binding private var image: NSImage?
+    @State private var blurReflection = true
+    
+    init(image: Binding<NSImage?>, reflectionDistance: CGFloat = -1.0, blurReflection: Bool = true, onTapGesture: (() -> ())? = nil) {
+        self._reflectionDistance = State<CGFloat>(initialValue: reflectionDistance)
+        self._blurReflection = State<Bool>(initialValue: blurReflection)
+        self._onTap = State<(() -> ())?>(initialValue: onTapGesture)
+        self._image = image
+    }
+    
+    var body: some View {
+        var image: NSImage!
+        if let posterImage = self.image {
+            image = posterImage
+        } else {
+            image = NSImage(named: "moviePosterPlaceholder")
+        }
+        var fadeImage = image.cgImage(forProposedRect: nil, context: nil, hints: nil)!
```

Comment:
Not sure how this line relates to the internet connection. Either we have a downloaded image, or we get the processed cgImage.
Either way this code is now removed because I switched to `SwiftUI.Image`.